portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -79,4 +79,28 @@ definition {
 			entryType = "${customElementType}");
 	}
 
+	@description = "Verify that remote app type IFrame can display property fields"
+	@priority = "4"
+	test IframeTypeCanDisplayProperFields {
+		property portal.acceptance = "true";
+
+		var customElementType = "Custom Element";
+
+		RemoteApps.goToRemoteAppsPortlet();
+
+		LexiconEntry.gotoAdd();
+
+		AssertNotVisible(locator1 = "RemoteAppsEntry#HTML_ELEMENT_INPUT");
```

Comment:
To make the test more robust, you'll probably want to verify that Iframe type is selected and if not then select Iframe type before asserting not visible.